app/javascript/article-form/components/Preview.jsx

```diff
@@ -41,76 +30,53 @@ function titleArea(previewResponse, version, articleState) {
     }
   }
 
-  const previewTitle = previewResponse.title || articleState.title || '';
-
-  let coverImageHTML = '';
-  if (coverImage.length > 0) {
-    coverImageHTML = (
-      <CoverImage
-        className="articleform__mainimage articleform__mainimagepreview"
-        imageSrc={coverImage}
-        imageAlt="cover"
-      />
-    );
-  }
+  const previewTitle = previewResponse.title || articleState.title;
```

Comment:
Any reason the `|| '';` is being omitted?

Reply:
not really 👍. i brought it back.